### PR TITLE
(#1753369) udev: enable U2F

### PR DESCRIPTION
**Pull request by dtardon**

### CODE REVIEW

File: rules/60-fido-id.rules [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="fido_id_end"
```
Comment (jsynacek) [inline rules/60-fido-id.rules:3]: Shouldn't this also say `SUBSYSTEM="hidraw"` to not match on every single device and needlessly execute a goto that does nothing?

Comment (dtardon) [reply rules/60-fido-id.rules:3]: I checked a handful of randomly picked rules files and they are all start with the same rule (modulo the goto name). So presumably it's not a problem?

Comment (jsynacek) [reply rules/60-fido-id.rules:3]: Well, they're all inefficient in the same way, in my opinion. But the impact is probably negligible, so let's leave it as it is.

**Review by jsynacek — APPROVED** [review]
LGTM.

**Pull request merged**